tests/test_qnode.py

```diff
@@ -621,21 +638,23 @@ def func(x, y):
     y = pnp.array(0.54, requires_grad=True)
 
     res = func(x, y)
+    tape = qml.workflow.construct_tape(func)(x, y)
 
-    assert isinstance(func.qtape, QuantumScript)
-    assert len(func.qtape.operations) == 3
-    assert len(func.qtape.observables) == 1
-    assert func.qtape.num_params == 2
+    assert isinstance(tape, QuantumScript)
+    assert len(tape.operations) == 3
+    assert len(tape.observables) == 1
+    assert tape.num_params == 2
 
-    expected = qml.execute([func.tape], dev, None)
+    expected = qml.execute([tape], dev, None)
     assert np.allclose(res, expected, atol=tol, rtol=0)
 
     # when called, a new quantum tape is constructed
-    old_tape = func.qtape
+    old_tape = tape
     res2 = func(x, y)
+    new_tape = qml.workflow.construct_tape(func)(x, y)
 
     assert np.allclose(res, res2, atol=tol, rtol=0)
-    assert func.qtape is not old_tape
+    assert new_tape is not old_tape
```

Comment:
Again, I think we can just delete the tape checking.  We know it's giving the correct answers, so that's what we should care about.